codecademy/introduction_to_classes.py

```diff
@@ -57,7 +57,11 @@ def display_order_history(self):
 
 class Triangle(object):
     def __init__(self, angle1, angle2, angle3):
-        assert type(angle1) == int and type(angle2) == int and type(angle3) == int, "Error type, please enter a valid value"
+        assert isinstance(angle1, (int, float)) and isinstance(angle2, (int, float)) and \
```

Comment:
`isinstance()` checks for superclasses too, remember?
```
>>> isinstance(True, int)
True
```
so better use `type()`
```python
        assert type(angle1) in (int, float) and type(angle2) in (int, float) and type(angle3) in (int, float)
```
or better loop it:
```python
for a in (angle1, angle2, angle3):
    assert type(a) in (int, float)
```